pom.xml

```diff
@@ -683,10 +683,6 @@
                      <relocation>
                        <pattern>google.iam</pattern>
                        <shadedPattern>${shadeBase}.google.iam</shadedPattern>
-                     </relocation>
```

Comment:
@sfc-gh-stakeda , do you know the original reason for the relocation and if it is okay to remove it?